conda_build/metadata.py

```diff
@@ -458,6 +458,13 @@ def binary_has_prefix_files(self):
                 raise RuntimeError("build/binary_has_prefix_files paths must use / as the path delimiter on Windows")
         return ret
 
+    def skip(self):
+        skip_expression = self.get_value('build/skip')
+        try:
+            return bool(eval(skip_expression, ns_cfg(), {}))
+        except:
+            return False
+
```

Comment:
I would not silently swallow any exceptions, because this would make typos hard to detect.  `False` should still be the default, however.